src/Preconditions/StringPreconditions.php

```diff
@@ -0,0 +1,105 @@
+<?php namespace Guardsman\Preconditions;
+
+use Guardsman\Exceptions\ArgumentNotAString;
+use Guardsman\Exceptions\StringTooLong;
+use Guardsman\Exceptions\StringTooShort;
+
+trait StringPreconditions
+{
+    /** @return mixed */
+    abstract public function getSubject();
+
+    /**
+     * @throws ArgumentNotAString if the subject is not a string.
+     *
+     * @return $this
+     */
+    public function isString()
+    {
+        if (!is_string($this->getSubject())) {
+            throw new ArgumentNotAString('Subject must be a string');
+        }
+
+        return $this;
+    }
+
+    /**
+     * @throws ArgumentNotNumeric if the limit is not numeric.
+     * @throws ValueTooSmall      if the limit is less than or equal to zero.
+     * @throws ArgumentNotAString if the subject is not a string.
+     * @throws StringTooLong      if the subject is longer than or equal to $limit.
+     *
+     * @return $this
+     */
+    public function isShorterThan($limit)
+    {
+        \Guardsman\check($limit)->isGreaterThan(0);
+        \Guardsman\check($this->getSubject())->isString();
+
+        if (mb_strlen($this->getSubject()) >= $limit) {
```

Comment:
FYI `mb_strlen()` way return false, I assume under invalid multibyte sequences. You way want to note that these methods are dependant on the value of the mbstring internal encoding.